core/src/main/java/org/geysermc/geyser/translator/inventory/chest/SingleChestInventoryTranslator.java

```diff
@@ -25,44 +25,100 @@
 
 package org.geysermc.geyser.translator.inventory.chest;
 
+import com.nukkitx.math.vector.Vector3f;
+import com.nukkitx.nbt.NbtMap;
+import com.nukkitx.protocol.bedrock.data.entity.EntityData;
+import com.nukkitx.protocol.bedrock.data.entity.EntityFlag;
+import com.nukkitx.protocol.bedrock.data.entity.EntityFlags;
 import com.nukkitx.protocol.bedrock.data.inventory.ContainerType;
+import com.nukkitx.protocol.bedrock.packet.AddEntityPacket;
+import com.nukkitx.protocol.bedrock.packet.BlockEntityDataPacket;
+import com.nukkitx.protocol.bedrock.packet.ContainerClosePacket;
+import com.nukkitx.protocol.bedrock.packet.ContainerOpenPacket;
+import com.nukkitx.protocol.bedrock.packet.RemoveEntityPacket;
+
```

Comment:
Remove this space - as of recent history, project imports have been grouped into non-Java imports and `java.*` imports.